pkg/provider/okta/okta.go

```diff
@@ -85,6 +85,12 @@ type mfaChallengeContext struct {
 	challengeResponseBody string
 }
 
+// mfaOption store the mfa position in response and mfa description
+type mfaOption struct {
```

Comment:
@hinling-sonder WDYT about extracting the `profile.authenticatorName` from the json and add it to this struct to have a human-readable MFA option in the prompt? I found it really helpful to have this instead of the full `profile` field.